### PR TITLE
T-41 Unit Testing for Creating Asset

### PR DESCRIPTION
**Pull request by DashaDrahun**
Unit Testing for Creating Asset

### CODE REVIEW

File: PortfolioKeeper.Tests/BusinessLogic/Helpers/AssetHelperTests.cs [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+using PortfolioKeeper.Helpers;
+using PortfolioKeeper.Models.ViewModels;
+
+namespace PortfolioKeeper.Tests.BusinessLogic.Helpers
+{
+    [TestFixture]
+    public class AssetHelperTests
+    {
+        private AssetHelper underTest;
+        private List<AssetPropertyTypeViewModel> allPropertyTypes;
```
Comment (AntonBessarabov) [inline PortfolioKeeper.Tests/BusinessLogic/Helpers/AssetHelperTests.cs]: I think you should delete this line. Or null this object in the setup method.

File: PortfolioKeeper.Tests/BusinessLogic/Helpers/AssetHelperTests.cs [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+                    Id = 1,
+                    Name = "Count"
+                }
+            };
+            var expectedResult = new AccountViewModel
+            {
+                DisplayCount = "DisplayCount",
+                ChildrenAssetTypeId = childrenTypeId
+            };
+            // act
```
Comment (AntonBessarabov) [inline PortfolioKeeper.Tests/BusinessLogic/Helpers/AssetHelperTests.cs]: Add empty line before `// act` , please

File: PortfolioKeeper.Tests/BusinessLogic/Services/AssetServiceTests.cs [hunk 3]
```diff
@@ -302,5 +332,112 @@ public void GetAsset_When_All_Valid_Should_Return_Valid_Dto()
...
+            {
+                new AssetPropertyType
+                {
+                    Id = 1,
+                    DisplayName = "ChildType",
+                    Name = "ChildType",
+                    DisplayOrder = 1,
+                    AssetTypeId = 1
+                }
+            };
```
Comment (AntonBessarabov) [inline PortfolioKeeper.Tests/BusinessLogic/Services/AssetServiceTests.cs]: Add empty line, please

File: PortfolioKeeper.Tests/BusinessLogic/Helpers/AssetHelperTests.cs [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+using NUnit.Framework;
+using PortfolioKeeper.Helpers;
+using PortfolioKeeper.Models.ViewModels;
+
+namespace PortfolioKeeper.Tests.BusinessLogic.Helpers
+{
+    [TestFixture]
+    public class AssetHelperTests
+    {
+        private AssetHelper underTest;
```
Comment (stas-litvinenka) [inline PortfolioKeeper.Tests/BusinessLogic/Helpers/AssetHelperTests.cs]: Interface

File: PortfolioKeeper.Tests/BusinessLogic/Helpers/AssetHelperTests.cs [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+
+            // act
+            var result = underTest.GetCar(allPropertyTypes);
+
+            // assert
+            result.Should().BeEquivalentTo(expectedResult);
+        }
+
+        [Test]
+        public void GetPropertyValuesFromTypes_When_Arguments_Are_Valid_Should_Return_PropertyValueViewModelCollection()
```
Comment (stas-litvinenka) [inline PortfolioKeeper.Tests/BusinessLogic/Helpers/AssetHelperTests.cs]: PropertyValueViewModelCollection-> PropertyValueViewModel

File: PortfolioKeeper.Tests/BusinessLogic/Helpers/AssetHelperTests.cs [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+        {
+            //arrange
+            var account = new AccountViewModel
+            {
+                ChildrenAssetTypeId = 1,
+                Count = 100,
+                Name = "BYN",
+                DisplayCount = "Количество"
+            };
+            List<AssetPropertyTypeViewModel> propertyTypes = new List<AssetPropertyTypeViewModel>
```
Comment (stas-litvinenka) [inline PortfolioKeeper.Tests/BusinessLogic/Helpers/AssetHelperTests.cs]: List<AssetPropertyTypeViewModel> -> var

File: PortfolioKeeper.Tests/BusinessLogic/Services/AssetServiceTests.cs [hunk 7]
```diff
@@ -192,10 +196,36 @@ public void GetAssetProperties_When_All_Valid_Should_Return_IEnumerable_Of_Asset
             var testUser = new AppUser();
             var value1A = new AssetPropertyValue { Id = 1, TypeId = 2, Value = "<value type=\"string\">BMW</value>" };
             var value2A = new AssetPropertyValue { Id = 2, TypeId = 3, Value = "<value type=\"int\">2016</value>" };
-            var value1B = new AssetPropertyValueDto { Id = 1, ValueType = typeof(string), Value = "BMW", PropertyType = new AssetPropertyTypeDto { DisplayOrder = 1 } };
-            var value2B = new AssetPropertyValueDto { Id = 2, ValueType = typeof(int), Value = "2016", PropertyType = new AssetPropertyTypeDto { DisplayOrder = 2 } };
+            var value1B = new AssetPropertyValueDto
```
Comment (stas-litvinenka) [inline PortfolioKeeper.Tests/BusinessLogic/Services/AssetServiceTests.cs]: rename to something more meaningful. e.g. propertyValue1, ....

File: PortfolioKeeper.Tests/BusinessLogic/Helpers/AssetHelperTests.cs [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+
+        [Test]
+        public void GetApartment_When_PropertyTypeCollection_Is_Not_Null_Should_Return_ApartmentViewModel()
+        {
+            // arrange
+            var allPropertyTypes = new List<AssetPropertyTypeViewModel>
+            {
+                new AssetPropertyTypeViewModel
+                {
+                    AssetTypeId = 1,
```
Comment (stas-litvinenka) [inline PortfolioKeeper.Tests/BusinessLogic/Helpers/AssetHelperTests.cs]: here you need only DisplayName and Name. the same below

**Pull request merged**